crates/ruff_db/src/files.rs

```diff
@@ -49,6 +52,8 @@ struct FilesInner {
 
     /// Lookup table that maps vendored files to the salsa [`File`] ingredients.
     vendored_by_path: FxDashMap<VendoredPathBuf, File>,
+
+    roots: std::sync::RwLock<FileRoots>,
```

Comment:
Can we add a doc comment here explaining what this is and why it's needed, similar to the above attributes?